reference-stores/next/pages/products/[handle].js

```diff
@@ -48,11 +48,11 @@ export async function getStaticProps({ params: { handle } }) {
   });
   const { page } = await resolvePageData({
     client: nacelleClient,
-    page: pages.edges[0]?.node
+    page: data.pages.edges[0]?.node
   });
   return {
     props: {
-      product: products.edges[0].node || null,
+      product: data.products.edges[0].node || null,
```

Comment:
Might be a good opportunity to use optional chaining with `.at`:

```suggestion
      product: data.products.edges.at(0)?.node || null,
```